clap_builder/src/builder/value_parser.rs

```diff
@@ -1828,7 +1828,7 @@ impl Default for FalseyValueParser {
     }
 }
 
-/// Parse bool-like string values, everything else is `true`
+/// Parse bool-like string values, everything else is an error
```

Comment:
Looks like this was mistakenly copied from `FalseyValueParser`.

The general assumption with value parsers is to error, so I think we should just drop everything after the `,`

Reply:
Yes, looks like this!
Updated.